doc/modules/ensemble.rst

```diff
@@ -681,7 +681,7 @@ approximated as follows:
 .. note::
 
   Briefly, a first-order Taylor approximation says that
-  :math:`l(z) \approx l(a) + (z - a) \frac{\partial l(a)}{\partial a}`.
+  :math:`l(z) \approx l(a) + (z - a) \frac{\partial l(a)}{\partial z}`.
```

Comment:
Nice for spotting this! I would even modify it for:
```suggestion
  :math:`l(z) \approx l(a) + (z - a) \frac{\partial l}{\partial z}(a)`.
```

This renders as:

$$l(z) \approx l(a) + (z - a) \frac{\partial l}{\partial z}(a)$$

Reply:
Extra change LGTM as well, committing the suggestion!